deb/build-debs.sh

```diff
@@ -176,9 +173,10 @@ cd "$DIRECTORY/mythtv"
 #these should always be parsed from the old changelog
 EPOCH=$(dpkg-parsechangelog -SVersion | sed 's/:.*//;')
 #actually bump the changelog up. don't include a git hash here right now.
+PACKAGING_HASH=$(git rev-parse --short HEAD)
```

Comment:
You can't move getting the PACKAGING_HASH to here since we have already changed directory into the mythtv git tree so it will either fail or get the wrong hash.

Reply:
Yes, you're right.Fixed with 07eac173d379387222bd8aac836bf3e918c91908

Reply:
The patch contained an error; correct fix is 74de93bae76086734531571115de8fa789854ec3

Reply:
Cherry-picked that as well, thanks.